JustSaying.Models.version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup Condition=" '$(MSBuildProjectName)' == 'JustSaying.Models' ">
-    <VersionPrefix>5.0.0</VersionPrefix>
-    <!-- <VersionSuffix></VersionSuffix> -->
+    <VersionPrefix>6.1.0</VersionPrefix>
```

Comment:
Given this is not a breaking change, it makes sense to keep this at 5 something. This package is deliberately versioned separately from the others.